src/env/Makefile.mk

```diff
@@ -29,25 +29,44 @@ endif BUILD_FC_BINDING
 
 if INSTALL_MPICXX
 bin_SCRIPTS += src/env/mpicxx
+if BUILD_ABI_LIB
+bin_SCRIPTS += src/env/mpicxx_abi
+endif BUILD_ABI_LIB
 endif INSTALL_MPICXX
 
 # create a local copy of the compiler wrapper that will actually be installed
 if BUILD_BASH_SCRIPTS
 src/env/mpicc: $(top_builddir)/src/env/mpicc.bash
 	cp -p $? $@
-src/env/mpicc_abi: $(top_builddir)/src/env/mpicc_abi.bash
+src/env/mpicc_abi: $(top_builddir)/src/env/mpicc.bash
 	cp -p $? $@
+	sed -e s/mpi_abi=no/mpi_abi=yes/ $@ > $@.abi
```

Comment:
Before I get a complaint for this `sed` + `cat` + `rm`, I don't know a Linux/macOS portable way to preserve mode form `cp -p`.

BTW, I felt tempted to fix all this code repetition in here. We can define lest say `shext = sh|bash` and to define a single set of recipes with `mpixxx.$shext`. Also, not sure what's the point of using `$?` rather than just `?<`.

Reply:
I think we can simply add an `-mpi-abi` option and install `mpicc-abi` as an alias to `mpicc`.